javav2/premium-ex.md

```diff
@@ -0,0 +1,11 @@
+//Add paths for premium examples here which will be consumed by workflow to update KB with quality code
+//Don't forget to add new line at the end of this file 
+
+## basics:
+/example_code/iotsitewise/src/main/java/com/example/iotsitewise/HelloSitewise.java
```

Comment:
This one seems to be pointing at the Hello example file, not the basics directory.

Reply:
Yess, but this is what was provided in the quip doc to me.